index.html

```diff
@@ -11,6 +11,9 @@
       <script src="https://kit.fontawesome.com/321e54ee43.js" crossorigin="anonymous"></script>
     </head>
     <body>
+
+    <!-- Here starts the header section -->
+
       <header>
         <nav class="navbar">
             <a class="logo-link"><p class="logo">Carrera<i  id=logo-dat class="fa-solid fa-circle-dot"></i>Programmer</p></a>
```

Comment:
- [ ] Kindly make sure your home page looks like in the Figma design with the same color, same format and same font size. You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma]
## Figma design

![image](https://user-images.githubusercontent.com/95297251/185690858-9882e76c-2e22-471d-b827-7f82448c8d36.png)


## your design
![image](https://user-images.githubusercontent.com/95297251/185690961-b2664356-d368-4040-a97f-1f30ad15fc45.png)